packer-provisioner-goss.go

```diff
@@ -22,6 +22,13 @@ import (
 const gossSpecFile = "/tmp/goss-spec.yaml"
 const gossDebugSpecFile = "/tmp/debug-goss-spec.yaml"
 
+type OsType string
+
+const (
+	Linux   = OsType("Linux")
+	Windows = OsType("Windows")
+)
```

Comment:
I think I understand why you did this, but I'm inclined to simplify this and just use a string and get rid of the const.  It will simplify the code below as well.  Thoughts?

Reply:
Which logic?  I think we still need all the checks in place as well as the `ToLower` to ensure the url is written properly. I could replace the `OsType` type and make it just a string?